modules/apps/object/object-service/src/main/java/com/liferay/object/internal/security/permission/resource/util/ObjectDefinitionResourcePermissionUtil.java

```diff
@@ -86,11 +86,14 @@ public static void populateResourceActions(
 				}));
 
 		resourceActions.populateModelResources(document);
-		resourceActions.populatePortletResource(
-			portletLocalService.getPortletById(
-				objectDefinition.getCompanyId(),
-				objectDefinition.getPortletId()),
-			classLoader, document);
+
+		if (objectDefinition.isPortlet() == true) {
```

Comment:
Since `isPortlet()` method returns a boolean, we don't need to check == true.
Do you agree? Could you improve this, please?